# Description a guide line with a lot of guide information for vehicle

uint64           id
Path             path
SpeedControl[]   speed
StopLine[]       stop
Boundary[]       boundary
bool             passable